VogtFactory/Test.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtFactory
+{
+    class Test
+    {
+        public static void Run()
+        {
+            RentalCar car = new Toyota();
```

Comment:
Ok so try this here:
```c#
var rcf = new RentalCarFactory();
var toyota = rcf .CreateRentalCar("Toyota");
var dodge = rcf .CreateRentalCar("Dodge");

toyota.DisplayCarInfo();
dodge.DisplayCarInfo();
```

THAT is how you, and the beauty of, using a factory.

Imagine you want to translate a user's input on the console to a c# class? A factory! The user passes in a STRING "Toyota", and you can get the c# class back in this way.